<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_hosting</class>
 <widget class="QWidget" name="widget_hosting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>439</y>
     <width>212</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>credit card:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_credit_card">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>159</y>
     <width>160</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlayout_guest_ages"/>
  </widget>
  <widget class="QPushButton" name="button_buy">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>499</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>buy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove_hosting">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>349</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>remove hosting</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>309</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlayout_chosen_hosting"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>439</y>
     <width>190</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Number of installments:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="combo_number_of_installments">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>441</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlayout_table_content"/>
  </widget>
  <widget class="QPushButton" name="button_choose_hosting">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>choose hosting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
